.github/workflows/release.yml

```diff
@@ -19,25 +19,24 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: 12
-    - name: yarn install
+    - name: npm install
```

Comment:
Why change this to npm?

Reply:
@billyvg I reverted it, wasnt necessary in the end